### PR TITLE
Add Atmo to Civo K3s Marketplace

### PR DESCRIPTION
**Pull request by cohix**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [ ] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [ ] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [ ] Tested that the application works with the proposed updates applied (including a screenshot)

### CODE REVIEW

File: atmo/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Atmo
```
Comment (saiyam1814) [inline atmo/manifest.yaml:2]: name in lower case 
Title  you can add as Atmo 
example https://github.com/civo/kubernetes-marketplace/blob/master/falco/manifest.yaml

File: atmo/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+version: 0.2.2
```
Comment (saiyam1814) [inline atmo/manifest.yaml:3]: version as string refer same example as above

**Comment by cohix** [comment]
@saiyam1814 Thanks for the review, the changes have been made!

**Comment by saiyam1814** [comment]
LGTM

**Pull request merged**